app/src/main/java/com/infomaniak/drive/ui/fileShared/FileSharedListFragment.kt

```diff
@@ -122,8 +162,50 @@ class FileSharedListFragment : FileListFragment() {
         )
     }
 
+    private fun downloadAllFiles() {
+        fileSharedViewModel.rootSharedFile.value?.let { file ->
```

Comment:
Add a little comment here to explain that here we are dealing with either a file or a folder of files